app/api/ada/index.js

```diff
@@ -557,12 +624,12 @@ export default class AdaApi {
     }
   }
 
-  async createTrezorWallet(
-    request: CreateTrezorWalletRequest
-  ): Promise<CreateTrezorWalletResponse> {
+  async createHardwareWallet(
+    request: CreateHardwareWalletRequest
+  ): Promise<CreateHardwareWalletResponse> {
     try {
-      Logger.debug('AdaApi::connectTrezor called');
-      const { walletName, publicMasterKey, deviceFeatures } = request;
+      Logger.debug('AdaApi::createHardwareWallet called');
+      const { walletName, walletHardwareInfo } = request;
```

Comment:
it’s not so obvious that contains the `publicMasterKey`. it should be extracted as well and then put back together in `cwHardwareInfo`